docs/zh/latest/admin-api.md

```diff
@@ -588,11 +588,11 @@ APISIX 的 Upstream 除了基本的负载均衡算法选择外，还支持对上
 
 `tls.client_cert/key` 可以用来跟上游进行 mTLS 通信。
 他们的格式和 SSL 对象的 `cert` 和 `key` 一样。
-这个特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)。
+这个特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6为-apache-apisix-构建-openresty)。
```

Comment:
Look like the doc site uses different way to generate the anchor, see https://apisix.apache.org/zh/docs/apisix/next/how-to-build/#步骤6：为-apache-apisix-构建-openresty

CC @juzhiyuan

Reply:
@leslie-tsang Hi, due to some different logics between github.com and https://docusaurus.io/ , we need to use a custom ID for those headers.

**Description**

We have one markdown file, see `https://github.com/apache/apisix/blob/master/docs/zh/latest/how-to-build.md#%E6%AD%A5%E9%AA%A46%E4%B8%BA-apache-apisix-%E6%9E%84%E5%BB%BA-openresty`

The URL is as follows:

![image](https://user-images.githubusercontent.com/2106987/139772763-0a56c173-0a31-4b83-8d1b-cf36d45789ca.png)

After it's transformed to Apache APISIX Website, the URL is as follows:

`https://apisix.apache.org/zh/docs/apisix/next/how-to-build/#%E6%AD%A5%E9%AA%A46%EF%BC%9A%E4%B8%BA-apache-apisix-%E6%9E%84%E5%BB%BA-openresty`

![image](https://user-images.githubusercontent.com/2106987/139772816-80a58ddc-4074-46c0-a033-c49f43fef35c.png)

**Issue**

We noticed that the URL is missing a `：`(in Chinese) in GitHub (see the first image).

**Guess**

1. The correct one is just like the second image on the website, `#xxxx` is a Slash.
2. GitHub maybe lose the needed `：`, it will break the Slash effect.

**Expect**

When we open links, it will navigate to the target section.

**Solution**

We could use custom header ids to fix this issue, see https://uhded.com/add-id-header-markdown .

Advantage: The sharable URL will be shorter and more semantic than the Chinese encoded style.

**What we do next**

We need to set custom ids (English or number) for those markdown headers :)

@leslie-tsang Would you like to do this?

Reply:
@juzhiyuan Sure.

Reply:
ok, if you have unclear points, please @ me 😄

Reply:
@leslie-tsang Hi, may I know the status of this PR? Do you need my help 😄